cime_config/config_component.xml

```diff
@@ -999,7 +999,7 @@
   <entry id="ESMF_VERBOSITY_LEVEL">
     <type>char</type>
     <valid_values>off,low,high,max</valid_values>
-    <default_value>low</default_value>
+    <default_value>off</default_value>
```

Comment:
Are you sure?  Do we still get a valid timing file with this setting?

Reply:
Yes valid timings are still obtained.